sycl/plugins/unified_runtime/CMakeLists.txt

```
@@ -53,14 +55,16 @@ endif()
 if(SYCL_PI_UR_USE_FETCH_CONTENT)
   include(FetchContent)
 
-  set(UNIFIED_RUNTIME_REPO "https://github.com/oneapi-src/unified-runtime.git")
-  # commit ec7982bac6cb3a6b9ed610cd6b7cb41fcbc780dc
-  # Merge: 62e6d2f9 5fb82924
+  set(UNIFIED_RUNTIME_REPO "https://github.com/martygrant/unified-runtime.git")
+
+  # commit d70bae454b0eacd4e273c5f697b42210fee9f706
+  # Merge: 612a263613b2 b01beb71f159
   # Author: Kenneth Benzie (Benie) <k.benzie@codeplay.com>
-  # Date:   Wed Nov 8 13:32:46 2023 +0000
-  #     Merge pull request #1022 from 0x12CC/l0_usm_error_checking_2
-  #     [UR][L0] Propagate OOM errors from `USMAllocationMakeResident`
-  set(UNIFIED_RUNTIME_TAG ec7982bac6cb3a6b9ed610cd6b7cb41fcbc780dc)
+  # Date:   Tue Nov 7 14:49:55 2023 +0000
+  #     Merge pull request #1026 from kbenzie/benie/test-conformance-sycl-library-dir
+  #     [CTS] Add UR_SYCL_LIBRARY_DIR CMake variable
+  # TODO: Don't merge yet; testing
+  set(UNIFIED_RUNTIME_TAG martin/moveNativeCPUAdapterToUR)
```

Now that  Please update https://github.com/oneapi-src/unified-runtime/pull/1059 has merged please pull in the latest changes from the sycl branch and update the `UNIFIED_RUNTIME_REPO` and `UNIFIED_RUNTIME_TAG` as follows.

```suggestion
  set(UNIFIED_RUNTIME_REPO "https://github.com/oneapi-src/unified-runtime.git")
  # commit 659d3f469faa99a886fa680a3d6d20449b109578
  # Merge: 192e9404 f94550b4
  # Author: Kenneth Benzie (Benie) <k.benzie@codeplay.com>
  # Date:   Tue Nov 14 16:45:24 2023 +0000
  #     Merge pull request #1059 from martygrant/martin/moveNativeCPUAdapterToUR
  #     [NATIVECPU] Move Native CPU adapter to UR.
  set(UNIFIED_RUNTIME_TAG 659d3f469faa99a886fa680a3d6d20449b109578)
```